package.json

```diff
@@ -1,8 +1,8 @@
 {
```

Comment:
We could also use the `files` field in `package.json` to whitelist the `dist` folder and other files, e.g. readme?